packages/hyperion-autologging/src/ALFlowletManager.ts

```diff
@@ -14,6 +14,7 @@ import { Flowlet } from "@hyperion/hyperion-flowlet/src/Flowlet";
  */
 export interface ALFlowletDataType {
   surface?: string;
+  region?: string;
```

Comment:
Would like to propose renaming these to map to the surface capabilities. If I understand correctly, each field should be a path of surfaces that filter for one of the surface capabilities. Then maybe something like `interactionSurface` and `mutationSurface`?

Reply:
hmm, I didn't change the name because we will not have one of these per each capability. Only these two for now. In future, the other ones that I had in mind will be applied to both. 
What if I rename `region` to `nonInteractiveSurface` so we know this is only about interactivity, otherwise for both types we will track mutation.